### DISCUSSION TITLE
I cant seem to download anything, (or run any commands actually).

### DISCUSSION THREAD

**Discussion by user4302** [Serene-Arc/bulk-downloader-for-reddit#448] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/32593782/121275664-d5373680-c8ea-11eb-951e-62969e9e4025.png)

i get the same output even if i run
python -m bdfr download 'D:<user>\Downloads\BDFR' --subreddit 'Python' -L 10

**Comment by Serene-Arc** [comment, marked as answer]
Please read the documentation. Python 3.9 or higher is required.

**Reply by user4302** [reply]
oh my, i thought i had v3.9+
apologies, and thank you.
